R/atom-plots.R

```diff
@@ -299,16 +301,17 @@ addLine <- function(data = NULL,
           shape = "legendLabels",
           fill = "legendLabels"
         ),
-        show.legend = TRUE
+        show.legend = NA

```

Comment:
@pchelle what is the impact of this change (here and below)?

Reply:
When using TRUE, I get the following weird point in the legend

```r
addLine(x=0)
```
![image](https://user-images.githubusercontent.com/45343665/165946380-f493bdfe-9bf2-43fb-b7f6-f6427f61f8a5.png)